cdap-common/bin/cdap

```diff
@@ -108,6 +109,7 @@ case ${1} in
     echo "    sdk          - Sends the arguments (start/stop/etc.) to the Standalone CDAP service on this host"
     echo
     fi
+    echo "    apply-pack   - Installs the UI Pack specified as an argument"
```

Comment:
Installs a CDAP Pack. There's a notion that this could be extended to other parts of CDAP (apps, etc)